test/e2e/storage/csi_volumes.go

```diff
@@ -84,7 +85,18 @@ var _ = Describe("E2E", func() {
 		testsuites.InitVolumeIOTestSuite,
 		testsuites.InitVolumeModeTestSuite,
 		testsuites.InitVolumesTestSuite,
-		testsuites.InitEphemeralTestSuite,
+	}
+	if k8sVersion := os.Getenv("TEST_KUBERNETES_VERSION"); k8sVersion != "" {
+		var major, minor int
+		if _, err := fmt.Sscanf(k8sVersion, "%d.%d", &major, &minor); err != nil {
+			framework.Logf("Failed to parse Kubernetes version '%s': %s", k8sVersion, err.Error())
```

Comment:
Can you skip this log message when `TEST_KUBERNETES_VERSION` is empty?

Reply:
Added log message in case of empty version string.